### PR TITLE
Feat: add quickstart video doc

### LINKED ISSUE #3905: [FEATURE] Embed quickstart YouTube video into quickstart guide
### Description

At the top of the quickstart docs it would be good to embed the YouTube video for it https://linkfree.eddiehub.io/docs/quickstart

YouTube video https://www.youtube.com/watch?v=05HEeCQSKRE

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Ertan222**
## Fixes Issue
Closes #3905 

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Idk this is a good position for video. 
Screenshot from 2023-01-28 04-28-26

## Note to reviewers

I can change position and width/height if this one looks not good. I couldn't decide it very well. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4148"></a>

### CODE REVIEW

File: components/QuickstartVideo.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default function QuickstartVideo() {
+    return (
+        <iframe width="410" 
+        height="220"
```
Comment (arpitghura) [inline components/QuickstartVideo.js:4]: I think you should make it responsive like larger view on larger size displays and smaller view on smaller display size.

Comment (Ertan222) [reply components/QuickstartVideo.js:4]: That's a good idea, going to add it. Thank you.

File: pages/docs/quickstart.mdx [hunk 2]
```diff
@@ -1,9 +1,14 @@
 import DocsLayout from "../../components/layouts/DocsLayout.js";
 import ClipboardCopy from "../../components/ClipboardCopy.js";
 import Link from "../../components/Link.js";
+import QuickstartVideo from "../../components/QuickstartVideo.js";
+
 
 ## Quickstart
 
+<QuickstartVideo>
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:9]: I think if it is going to be a new component, the video url should be a prop so we can reuse the component in other areas in the docs, as we have more doc videos coming out

File: components/QuickstartVideo.js (hunk 1 above)
Comment (eddiejaoude) [reply components/QuickstartVideo.js:4]: Good idea, TailWind has that built in https://tailwindcss.com/docs/aspect-ratio

File: pages/docs/quickstart.mdx (hunk 2 above)
Comment (Ertan222) [reply pages/docs/quickstart.mdx:9]: I thought that for future use of the same component. Thank you for feedback.

**Comment by Ertan222** [comment]
I am so sorry for multiple different review request. I thought can request review from multiple people. Didn't mean to disturb anyone

**Review by arpitghura — APPROVED** [review]
Looks Good

**Comment by Ertan222** [comment]
I am new to responsive design and ı saw that, quickstart images is not responsive as much other doc images. Is that normal? I checked image dimensions and they are almost same (a little bit diff but not so much).  

https://user-images.githubusercontent.com/62712555/215660972-23a7e78b-d6ad-416d-aed6-d1d65157d498.mp4

**Comment by eddiejaoude** [comment]
There is some responsiveness, but improvements are needed, these can come in another pull request

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**